webplugin/js/app/zendesk-chat-service.js

```diff
@@ -44,47 +94,34 @@ function ZendeskChatService() {
 
             zChat.sendFile(file, function (err, data) {
                 if (err) {
-                    window.console.log("Error while sending file : ",err);
+                    console.log("Error while sending file : ", err);
```

Comment:
what's the correct way, in last PR, Raj added window. to every console.log and in this PR you are removing all window.
which one is the standard and consistent way to use it?

Reply:
cc: @raj5036